lib/concurrent/executor/executor.rb

```diff
@@ -59,6 +59,38 @@ def handle_fallback(*args)
     def serialized?
       false
     end
+
+    def auto_terminate?
+      !! @auto_terminate
+    end
+
+    protected
+
+    def enable_at_exit_handler!(opts = {})
+      if opts.fetch(:stop_on_exit, true)
```

Comment:
I would argue that custom thread creation is more in a land of advanced users so it might may sense to make the default `false`. Hm, but that may be subjective.